### PR TITLE
Upgrade to qunit 5

### PR DESCRIPTION
**Pull request by candunaj**
Currently ember-simple-auth is using ember-qunit 4 which does not support ember 4. 
I have upgraded ember-simple-auth to ember-qunit 5. ember-qunit 5 does not support ember 3.0 so I have removed it from ci.

### CODE REVIEW

File: packages/ember-simple-auth/config/ember-try.js [hunk 1]
```diff
@@ -14,21 +14,6 @@ module.exports = function() {
     return {
       useYarn: true,
       scenarios: [
-        {
-          name: 'ember-3.0',
```
Comment (BobrImperator) [inline packages/ember-simple-auth/config/ember-try.js:18]: I believe we already spoke about this, but just to make sure - we have to drop the ember 3.0 in order to use this qunit version?

Qunit 5 is the one compatible with embroider, right?

Comment (candunaj) [reply packages/ember-simple-auth/config/ember-try.js:18]: ember-qunit 4 does not support ember 4. I have upgraded ember-qunit to 5 to be able to support ember-4.

Comment (BobrImperator) [reply packages/ember-simple-auth/config/ember-try.js:18]: Got it 🚀

**Comment by marcoow** [comment]
This can only go out with 5.0.0 so we need to put this on hold for now…

**Comment by marcoow** [comment]
@candunaj I'd like to get something like #2353 working first tbh

**Review by BobrImperator — APPROVED** [review]

**Pull request merged**